### DISCUSSION TITLE
CMT2300A EBYTE E49-900M20S - Antenne erforderlich?

### DISCUSSION THREAD

**Discussion by mc-0815** [tbnobody/OpenDTU#909] [Q&A]
Hi,

short question for the audience. Do I really need an external antenna when using CMT2300A EBYTE E49-900M20S with a HMS-1600? And if so which one can you propose.

The data sheet of the E49-900M20S shows an internal one and a connector for an external one.

Maybe someone has already experiences that he or she can share.

Many thanks.

**Comment by Sandern2712** [comment]
Hallo,

Kurze Frage an das Publikum. Benötige ich wirklich eine externe Antenne, wenn ich CMT2300A EBYTE E49-900M20S mit einem HMS-1600 verwende? Und wenn ja, welches können Sie vorschlagen?

Das Datenblatt des E49-900M20S zeigt einen internen und einen Anschluss für einen externen.

Vielleicht hat jemand bereits Erfahrungen gemacht, die er oder sie teilen kann.

Vielen Dank.
